cockroachcloud/backups-page.md

```diff
@@ -69,19 +71,95 @@ For each table in the database, the following details display:
 
 - The number of **Rows** captured in the backup
 
-## Back up a self-hosted CockroachDB cluster and restore into a CockroachCloud cluster
+## Ways to restore data
+
+[Console Admin](console-access-management.html#console-admin) can perform the following from the Console:
+
+- [Restore a database](#restore-a-database)
+- [Restore a table](#restore-a-table)
+
+Additional ways to restore data:
+
+- [Back up a self-hosted CockroachDB cluster and restore into a CockroachCloud cluster](#back-up-a-self-hosted-cockroachdb-cluster-and-restore-into-a-cockroachcloud-cluster)
+- [Back up and restore data manually](#back-up-and-restore-data-manually)
+
+### Restore a database
+
+To restore a database:
+
+1. Find the cluster backup containing the database you want to restore, and click the number in the corresponding **Databases** column.
+
+    The **Databases** view displays.
+
+1. Click **Restore** for the database you want to restore.
+
+    The **Restore database** module displays with backup details.
+
+1. In the **Restore to** field, enter the name of the destination database.
+
+    {{site.data.alerts.callout_info}}
+    [Resolve any naming conflicts](#resolve-a-database-naming-conflict) by using `DROP` or `RENAME` on the existing database. If you enter a unique name in the **Restore to** field, a new database will be created.
+    {{site.data.alerts.end}}  
+
+1. Select any of the **Dependency options** to skip. You can:
+    - **Skip missing foreign keys**, which will remove [foreign key](../stable/foreign-key.html) constraints before restoring.
```

Comment:
Dumb question: does it remove FKs?  or just "skip missing ones" as the other text says?

Reply:
if it does remove FKs, does it remove some or all? we should say which, I think